packages/blocks/src/root-block/widgets/linked-doc/index.ts

```diff
@@ -232,6 +222,25 @@ export class AffineLinkedDocWidget extends WidgetComponent {
     this.handleEvent('keyDown', this._onKeyDown);
     this.handleEvent('compositionEnd', this._onCompositionEnd);
   }
+
+  override render() {
+    if (this._show.value === 'none') return nothing;
+
+    return html`<blocksuite-portal
+      .shadowDom=${false}
+      .template=${choose(
+        this._show.value,
+        [['desktop', this._renderDesktopLinkedDocPopover]],
+        () => html`${nothing}`
```

Comment:
Can return nothing directly?

Reply:
This line is necessary because the third parameter of `choose` is not optional.

Reply:
Well, I misunderstood your comment. If return `nothing` directly, a type error will throw because `nothing` is not same as `TemplateResult`